src/Service/ReplayImportService.php

```diff
@@ -180,11 +194,26 @@ class ReplayImportService
     /** @var array<int, string> A map of flag IDs to flag abbreviations */
     private $flagIDs;
 
-    public function __construct(EntityManagerInterface $em, LoggerInterface $logger, MapThumbnailWriterService $thumbnailWriterService)
+    /** @var array<string, PlayerMovementGrid> A map of callsigns to their respective movement grids */
+    private $currPlayersHeatMap;
+
+    /** @var int The size of the Heatmap */
+    private $heatMapSize;
+
+    /** @var int The size of the World */
+    private $worldSize;
```

Comment:
```suggestion
    /** @var int The number of cells in a single row of the heatmap */
    private $heatMapSize;

    /** @var int The size of the world in World Units (wu) */
    private $worldSize;
```

Let's change the doc descriptions for these variables